marlowe-test/src/Spec/Marlowe/Semantics/Arbitrary.hs

```diff
@@ -350,19 +351,17 @@ randomTokenNames =
   ]
 
 instance Arbitrary TokenName where
-  arbitrary = arbitraryFibonacci randomTokenNames
+  arbitrary = sized \size -> elements $ takeWhile ((<= size) . fromInteger . lengthOfByteString . unTokenName) randomTokenNames
```

Comment:
Use size parameter to control length of chosen value rather than Fibonacci frequencies